slides/lectures/01/processor/slides.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: section
+---
+# What is a microprocessor?
+
+---
+layout: two-cols
+---
+# Microcontroller (MCU)
+controls hardware
+
+- low operating frequency (MHz)
+- a lot of I/O ports
+- runs simple software
+- controls hardware
+- does not require an Operating System
+
+<img src="/processor/pico.jpg" class="m-10 h-30">
+
+
+:: right ::
+
+# Microprocessor (CPU)
+computes
+
```

Comment:
General purpose, for PC & workstations

Reply:
Debatable here, some devices do have big chips for computing, like intelligent door handles, they for sure use a CPU.